Makefile

```diff
@@ -5,16 +5,15 @@ BIN_OUTPUT_PATH = bin
 TOOL_BIN = bin/gotools/$(shell uname -s)-$(shell uname -m)
 UNAME_S ?= $(shell uname -s)
 
-.PHONY: build
-build:
-	rm -f $(BIN_OUTPUT_PATH)/lorawan
-	CGO_LDFLAGS="$$CGO_LDFLAGS $(CGO_BUILD_LDFLAGS)" go build $(GO_BUILD_LDFLAGS) -o  $(BIN_OUTPUT_PATH)/lorawan main.go
+lorawan:
+	rm -f lorawan
+	CGO_LDFLAGS="$$CGO_LDFLAGS $(CGO_BUILD_LDFLAGS)" go build $(GO_BUILD_LDFLAGS) -o $@ main.go
 
-module.tar.gz: sx1302 build
+module.tar.gz: lorawan first_run.sh meta.json
```

Comment:
let's not mess with the makefile too much every pr going forward.

Reply:
the module.tar.gz has to include these things for the first_run to work but yeah this should be the last PR with makefile changes